cpp/src/arrow/compute/exec.cc

```diff
@@ -90,14 +90,29 @@ void PrintTo(const ExecBatch& batch, std::ostream* os) {
     if (value.is_scalar()) {
       *os << "Scalar[" << value.scalar()->ToString() << "]\n";
       continue;
```

Comment:
Should we remove this?

Reply:
You mean remove the `continue` statement? I suppose we could.